### PR TITLE
Create jain2020contrastive.markdown

### PR DESCRIPTION
**Pull request by reshinthadithyan**
Added Contrastive Code Representation Learning.

### CODE REVIEW

File: _publications/jain2020contrastive.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: publication
+title: "Contrastive Code Representation Learning"
+authors: Paras Jain, Ajay Jain, Tianjun Zhang, Pieter Abbeel, Joseph E. Gonzalez, Ion Stoica
+conference: ICML
```
Comment (mallamanis) [inline _publications/jain2020contrastive.markdown:5]: @reshinthadithyan the paper looks like a NeurIPS submission
```suggestion
conference: 
```

File: _publications/jain2020contrastive.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+year: 2020
+bibkey: jain2020contrastive
+additional_links:
+   - {name: "ArXiV", url: "https://arxiv.org/abs/2007.04973"}
+   - {name: "Website", url: "https://parasj.github.io/contracode/"}
+   - {name: "GitHub", url : "https://github.com/parasj/contracode"}
+tags: ["Contrastive Learning"]
```
Comment (mallamanis) [inline _publications/jain2020contrastive.markdown:12]: I am trying to keep a small set of tags, what about using some of the existing ones?
```suggestion
tags: ["representation", "pretraining"]
```

**Comment by reshinthadithyan** [comment]
@mallamanis, My bad. It's not ICML. Yes, it's good to have the tags has you've mentioned. Since Contrastive Learning is gaining attraction, we might soon need to have one _self-supervised_ tag.

**Comment by mallamanis** [comment]
@reshinthadithyan adding a "self supervised" tag makes sense. I'll keep that in mind in the future, but if you have time, feel free to take a pass over existing papers and add it 🚀

**Comment by reshinthadithyan** [comment]
Sure, I'll try adding it in the coming week.

**Pull request merged**